src/org/labkey/test/tests/MessagesLongTest.java

```diff
@@ -424,6 +439,37 @@ public void testSteps()
         assertTextPresent("The following new posts were made yesterday");
     }
 
+    @Test
+    public void testDailyDigestMessage()
```

Comment:
This is really testing the groups functionality. The daily digest is tangential.